src/sql-parser/src/parser.rs

```diff
@@ -2629,14 +2629,22 @@ impl<'a> Parser<'a> {
         let if_not_exists = self.parse_if_not_exists()?;
         let name = self.parse_item_name()?;
 
-        // Webhook Source, which works differently than all other sources.
-        if self.peek_keywords(&[FROM, WEBHOOK]) {
-            return self.parse_create_webhook_source(name, if_not_exists);
-        }
-
         let (col_names, key_constraint) = self.parse_source_columns()?;
+
+        // For webhook sources we require `IN CLUSTER`, so track the position for an error message.
+        let cluster_err = self.expected(self.peek_pos(), "IN CLUSTER", self.peek_token());
         let in_cluster = self.parse_optional_in_cluster()?;
         self.expect_keyword(FROM)?;
```

Comment:
What I'm trying to do here is represent "in_cluster is optional for all sources, but required for webhook sources". If folks have a better idea on how to do this it would be much appreciated!